src/Components/DrillPage.js

```diff
@@ -101,7 +101,14 @@ export const DrillPage = (props) => {
           <Text style={styles.descriptionText}>{drill.description}</Text>
         </View>
       </View>
-      <View ref={firstDrill} style={styles.animation}>
+      <View style={styles.lines} />
```

Comment:
What about fitness drills?

I think we have 2 choices:
- provide an `inGame` description for them? (is it interesting?)
- display it only for Frisbee drills?

Reply:
I don't think it's interesting in the description. I think to display it only on frisbee drills